dbt/models/mart/all_available_listings.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(materialized='table') }}
```

Comment:
This is a new table that we hadn't previously discussed. My rationale was that the stg_listings model should be very close to the raw form so it has all the records (~35 million). But we only need listings for which we can make a city join (~14 million). In turn, this table is used by the fact table, the listings dimension table, and the analytics listings flag table, so it seemed like it would be a good choice to make it a table (vs view). Let me know if this is a good choice.